2025-02-25 01:08:58.406 [info] Started local extension host with pid 19216.
2025-02-25 01:08:58.549 [error] Extension 'ms-python.debugpy CANNOT USE these API proposals 'portsAttributes, debugVisualization, contribViewsWelcome'. You MUST start in extension development mode or use the --enable-proposed-api command line flag
2025-02-25 01:08:58.549 [error] Extension 'ms-python.python CANNOT USE these API proposals 'contribEditorContentMenu, quickPickSortByLabel, testObserver, quickPickItemTooltip, terminalDataWriteEvent, terminalExecuteCommandEvent, codeActionAI, notebookReplDocument, notebookVariableProvider'. You MUST start in extension development mode or use the --enable-proposed-api command line flag
2025-02-25 01:08:58.566 [error] [ms-python.python]: editor/content is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribEditorContentMenu"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api ms-python.python
2025-02-25 01:09:00.111 [error] An unknown error occurred. Please consult the log for more details.
2025-02-25 01:09:00.843 [error] Error while getting the latest version for the extension kevinrose.vsc-python-indent. Cannot read properties of undefined (reading 'replace')
2025-02-25 01:09:00.843 [error] Error while getting the latest version for the extension ms-python.debugpy. Cannot read properties of undefined (reading 'replace')
2025-02-25 01:09:00.844 [error] Error while getting the latest version for the extension ms-python.python. Cannot read properties of undefined (reading 'replace')
2025-02-25 01:09:03.026 [info] [perf] Render performance baseline is 19ms
2025-02-25 01:27:28.853 [error] An unknown error occurred. Please consult the log for more details.
2025-02-25 01:37:26.669 [info] Extension host (LocalProcess pid: 19216) is unresponsive.
2025-02-25 01:38:19.045 [error] Extension host (LocalProcess pid: 19216) terminated unexpectedly. The following extensions were running: ms-python.python, vscode.emmet, KevinRose.vsc-python-indent, ms-python.debugpy, vscode.git-base, vscode.git, vscode.github, vscode.debug-auto-launch, vscode.merge-conflict, vscode.github-authentication, vscode.configuration-editing, vscode.json-language-features, vscode.typescript-language-features, vscode.npm
2025-02-25 01:38:19.047 [info] Automatically restarting the extension host.
2025-02-25 01:38:19.122 [info] Started local extension host with pid 22885.
2025-02-25 01:38:19.987 [error] No registered selector for ID: ms-vscode.npm-command
2025-02-25 01:38:20.967 [error] An unknown error occurred. Please consult the log for more details.
2025-02-25 01:59:32.123 [error] UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value: Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-25 01:59:33.934 [error] UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value: Error: UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-25 01:59:34.964 [error] UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value: Error: UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-25 01:59:35.395 [error] UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value: Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-25 01:59:36.521 [error] UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value: Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-25 01:59:37.521 [error] UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value: Error: UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-25 02:05:49.819 [info] Extension host (LocalProcess pid: 22885) is unresponsive.
2025-02-25 02:06:10.046 [error] Extension host (LocalProcess pid: 22885) terminated unexpectedly. The following extensions were running: ms-python.python, vscode.emmet, KevinRose.vsc-python-indent, ms-python.debugpy, vscode.git-base, vscode.git, vscode.github, vscode.debug-auto-launch, vscode.merge-conflict, vscode.github-authentication, vscode.configuration-editing, vscode.json-language-features, vscode.typescript-language-features, vscode.npm
2025-02-25 02:06:10.048 [info] Automatically restarting the extension host.
2025-02-25 02:06:10.120 [info] Started local extension host with pid 25920.
2025-02-25 02:06:11.123 [error] No registered selector for ID: ms-vscode.npm-command
2025-02-25 02:06:11.906 [error] An unknown error occurred. Please consult the log for more details.
2025-02-25 02:33:44.526 [info] Extension host (LocalProcess pid: 25920) is unresponsive.
2025-02-25 02:34:23.669 [error] Error: Aborted onWillSaveTextDocument-event after 1750ms
    at vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:737:10013
